.github/workflows/release-phar.yml

```
@@ -29,18 +27,33 @@ jobs:
+        uses: softprops/action-gh-release@v2
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          name: asd.phar
+
+      - name: Release
+        id: Release
+        uses: softprops/action-gh-release@v2
+
+      - name: Attach artifacts to release
+        uses: actions/upload-release-asset@v1
+        with:
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 54: did not find expected key

(syntax-check)